### DISCUSSION TITLE
Workflows update rejection - aborting branch.

### DISCUSSION THREAD

**Discussion by adamstrawson** [renovatebot/renovate#19390] [Request Help] [answered]
I'm trying to understand why this rejection is occurring, there isn't anything more logged besides this single line.

`INFO: Workflows update rejection - aborting branch. (repository={org}/{repo} branch=renovate/actions-checkout-3.x)` 

We're seeing this across a number of repositories, all which the same `renovate.json` below:

`renovate.json`
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": [
        "minor",
        "patch",
        "pin",
        "digest"
      ],
      "automerge": true
    }
  ]
}
```

We're running self-hosted Renovate (on Kubernetes), version `34.55.0`)

**Comment by viceice** [comment, marked as answer]
are you running against github or github enterprise? Then your renovate credentials are missing workflow permissions

- https://docs.renovatebot.com/security-and-permissions/#global-permissions

**Reply by adamstrawson** [reply]
That did the trick, thanks @viceice

**Comment by fhg-isi** [comment]
Uninstalling renovate GitHub App and installing the recent version somehow solved the permission issue for me.
